pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -551,41 +550,42 @@ def test_reindex_api_equivalence(self):
         for res in [res2, res3]:
             tm.assert_frame_equal(res1, res)
 
-    def test_align(self):
-        af, bf = self.frame.align(self.frame)
-        assert af._data is not self.frame._data
+    def test_align(self, float_frame, int_frame, float_string_frame,
```

Comment:
if you are adding many fixtures to a test, would rather use a parametrization of those fixtures, or create separate tests